packages/core/src/com/helpers.ts

```diff
@@ -23,3 +23,11 @@ export class MultiCounter {
         return `${ns}${this.ids[ns]++}`;
     }
 }
+
+const undefinedArgPlaceholder = '__|UND_ARG_PLA|__';
+
+export const serializeApiCallArguments = (args: unknown[]): unknown[] =>
```

Comment:
Add a comment or a link to the issue explaining why this is necessary.

Reply:
@alisey fixed